packages/@aws-cdk/aws-iot/README.md

```diff
@@ -72,3 +74,12 @@ const topicRule = new iot.TopicRule(this, 'TopicRule', {
 });
 topicRule.addAction(new actions.LambdaFunctionAction(func))
 ```
+
+If you wanna make the topic rule disable, add property `enabled: false` as following:
```

Comment:
"If you wanna make the topic rule disable" -> "If you want to make the topic rule disasbled"